### ISSUE TITLE
Focus representation should remain consistent throughout page

### ISSUE THREAD

**Issue by maxkadel** [pulibrary/allsearch_frontend#485]
### Steps to recreate
Currently, the focus representation in the menu bar is orange, and then when you tab down to the list of sections (Catalog, Articles+, etc.), it switches to black. The representation should remain consistent.

### User impact
Low

### Recommended fix
Maintain the focus representation as an orange box (don't switch to black)

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...
